modules/tobira/src/main/java/org/opencastproject/tobira/impl/Harvest.java

```diff
@@ -89,8 +101,9 @@ static Jsons.Obj harvest(
     // Retrieve episodes from index.
     final SearchSourceBuilder q = new SearchSourceBuilder().query(
             QueryBuilders.boolQuery()
-                .must(QueryBuilders.rangeQuery(SearchResult.MODIFIED_DATE).gte(since.getTime()))
-                .must(QueryBuilders.termQuery(SearchResult.TYPE, SearchService.IndexEntryType.Episode)))
+                    .must(QueryBuilders.termQuery(SearchResult.ORG, org))
+                    .must(QueryBuilders.rangeQuery(SearchResult.MODIFIED_DATE).gte(since.getTime()))
+                    .must(QueryBuilders.termQuery(SearchResult.TYPE, SearchService.IndexEntryType.Episode)))
```

Comment:
```suggestion
                .must(QueryBuilders.termQuery(SearchResult.ORG, org))
                .must(QueryBuilders.rangeQuery(SearchResult.MODIFIED_DATE).gte(since.getTime()))
                .must(QueryBuilders.termQuery(SearchResult.TYPE, SearchService.IndexEntryType.Episode)))
```

Four extra spaces which don't match the surrounding code